src/bindings/overlay.js

```diff
@@ -130,7 +141,10 @@ function overlay(sourceElement, translationElement) {
   // cleared if a new language doesn't use them; https://bugzil.la/922577
   if (translationElement.attributes) {
     for (k = 0, attr; (attr = translationElement.attributes[k]); k++) {
-      if (isAttrAllowed(attr, sourceElement)) {
+      if (isAttrAllowed({
```

Comment:
II couldn't go for it because HTML doesn't have attribute namespaces.

On IRC we agreed to use target element's namespace